security-read-me.Rmd

```diff
@@ -0,0 +1,25 @@
+---
+title: "security-read-me2"
+author: "cherylachary"
+date: "19/05/2021"
+output: html_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+## What Cheryl did to set up her security to safely store secret codes and keep data and key items off the internet
+
+- Forked [inform-health-informatics/datascibc-noobie](https://github.com/inform-health-informatics/datascibc-noobie) and added to [my] (https://github.com/tagimoucia-cheryl?tab=repositories)repository.
+- Cloned repository onto my datascience desktop into directory: B:/datascibc-noobie/public-datascibc-noobie
+- saved [example-config](https://github.com/tagimoucia-cheryl/datascibc-noobie-1/tree/feature/security-basics)as .Renviron in  directory on data science desktop (B:/datascibc-noobie/public-datascibc-noobie)
+- I populated UDS detail with that used to access EMAP via dbForge 
+- I have detailed the UCH proxy address as: 
+_http_proxy=http://www-cache-n.xuclh.nhs.uk:3128//_
+_HTTP_PROXY=http://www-cache-n.xuclh.nhs.uk:3128/_
+_https_proxy=http://www-cache-n.xuclh.nhs.uk:3128/_
+_HTTPS_PROXY=http://www-cache-n.xuclh.nhs.uk:3128/_
+
```

Comment:
**Option 2 ; for keeping secrets save**

Step 1 : Make a R file with your secrets. I will frequently call this file secrets. R ( The following is for when your using EMAP)

host = "xxxxxxxxxxxxxxxx"
port = ****
user ="johndoe"
password = "12456789"
dbname = "uds"

Step 2 : Go to the gitignore file  and enter  following information 
                    - secrets.R 
                     -Rhistory

Step 3 : Go to the R script you are working on and enter the following 
                - source("secrets’")